### PR TITLE
BUGZ-765: Snaps Photos are Dark

### PR DESCRIPTION
**Pull request by annabrewer**
The way I restructured the gamma correction when I was fixing it on Android made snapshots in desktop too dark because it messed with the composite pipeline. I've changed it back to a similar logic as before, where the SRGB -> RGB transformation on desktop happens in renderFromTexture instead of compositeScene.
https://highfidelity.atlassian.net/browse/BUGZ-765

### CODE REVIEW

File: libraries/display-plugins/src/display-plugins/Basic2DWindowOpenGLDisplayPlugin.cpp [hunk 1]
```diff
@@ -109,11 +109,25 @@ bool Basic2DWindowOpenGLDisplayPlugin::internalActivate() {
     return Parent::internalActivate();
 }
 
+gpu::PipelinePointer Basic2DWindowOpenGLDisplayPlugin::getRenderTexturePipeline() {
+#if defined(Q_OS_ANDROID)
+    return _linearToSRGBPipeline;
+#else
+
+#ifndef USE_GLES
```
Comment (SamGondelman) [inline libraries/display-plugins/src/display-plugins/Basic2DWindowOpenGLDisplayPlugin.cpp]: Why is this necessary?  Have we tried it with USE_GLES on desktop?

File: libraries/display-plugins/src/display-plugins/OpenGLDisplayPlugin.cpp [hunk 2]
```diff
@@ -975,3 +947,17 @@ void OpenGLDisplayPlugin::copyTextureToQuickFramebuffer(NetworkTexturePointer ne
...
+gpu::PipelinePointer OpenGLDisplayPlugin::getCompositeScenePipeline() {
+    return _drawTexturePipeline;
+}
+
+// Added this to allow desktop composite framebuffer to be RGBA while mobile is SRGBA, so that tone mapping looks right on both platforms
+// Overridden by Basic2DWindowDisplayPlugin to achieve this
+// FIXME: Eventually it would be ideal to have both framebuffers be of the same type
+gpu::Element OpenGLDisplayPlugin::getCompositeFBColorSpace() {
+    return gpu::Element::COLOR_RGBA_32;
+}
```
Comment (SamGondelman) [inline libraries/display-plugins/src/display-plugins/OpenGLDisplayPlugin.cpp]: It looks like VS formatted this file and the .h for you.  Can we undo that to unclutter these changes?

Also, is getCompositeFBColorSpace unused now?

Comment (annabrewer) [reply libraries/display-plugins/src/display-plugins/OpenGLDisplayPlugin.cpp]: 1. Sure
2. No, I just forgot to add it back into updateCompositeFramebuffer, will do in a sec

File: libraries/display-plugins/src/display-plugins/Basic2DWindowOpenGLDisplayPlugin.cpp (hunk 1 above)
Comment (annabrewer) [reply libraries/display-plugins/src/display-plugins/Basic2DWindowOpenGLDisplayPlugin.cpp]: I was just preserving the logic from before. Should probably test that.

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20081/HighFidelity-Beta-PR15815-70fbe84.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20081/HighFidelity-Quest-Beta-PR15815-70fbe84.apk)

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20081/HighFidelity-Beta-PR15815-70fbe84.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20081/HighFidelity-Beta-Interface-PR15815-70fbe84.exe

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20085/HighFidelity-Beta-PR15815-17163a1.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20085/HighFidelity-Beta-Interface-PR15815-17163a1.exe

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20085/HighFidelity-Beta-PR15815-17163a1.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20085/HighFidelity-Quest-Beta-PR15815-17163a1.apk)

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20097/HighFidelity-Beta-PR15815-8c3ddc0.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20097/HighFidelity-Quest-Beta-PR15815-8c3ddc0.apk)

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20097/HighFidelity-Beta-PR15815-8c3ddc0.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20097/HighFidelity-Beta-Interface-PR15815-8c3ddc0.exe

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20120/HighFidelity-Beta-PR15815-792cef1.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20120/HighFidelity-Quest-Beta-PR15815-792cef1.apk)

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20120/HighFidelity-Beta-PR15815-792cef1.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20120/HighFidelity-Beta-Interface-PR15815-792cef1.dmg

**Comment by SamGondelman** [comment]
build this please

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20122/HighFidelity-Beta-PR15815-792cef1.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20122/HighFidelity-Quest-Beta-PR15815-792cef1.apk)

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20122/HighFidelity-Beta-PR15815-792cef1.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20122/HighFidelity-Beta-Interface-PR15815-792cef1.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20122/HighFidelity-Beta-PR15815-792cef1.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20122/HighFidelity-Beta-Interface-PR15815-792cef1.exe
